src/components/CommentsList.jsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react";
+import { getComments } from "../utils/api";
+import { useParams } from "react-router-dom";
+import Comment from "./Comment";
+
+const CommentList = () => {
+    const {article_id} = useParams();
+    const [comments, setComments] = useState();
+    const [isLoading, setIsLoading] =useState(true);
+
+    useEffect(() => {
+        getComments(article_id).then((res) => {
+            setComments(res.data.comments)
+            setIsLoading(false)
+        })
+    }, []);
+
+    if (isLoading) {
+        return <p>Loading...</p>
+    }
+
+    return (
+        <section>
+            <h4>Commnets</h4>
+            {comments.map((comment) => {
+                return <Comment key={comment.comment_id} comment={comment}/>
+            })}
+        </section>
+    )
+};
```

Comment:
Great :) 
The only thing I would add is a conditional message for if the article has no comments.